### PR TITLE
Implemented menu screen

### PR DESCRIPTION
**Pull request by princessdharmy**

### CODE REVIEW

File: app/src/main/java/com/am/fdamilola/bankingapp/activities/MenuActivity.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.am.fdamilola.bankingapp.activities;
+
+import android.os.Bundle;
+
+import com.am.fdamilola.bankingapp.R;
+import com.am.fdamilola.bankingapp.base.classes.BankingAppBaseActivity;
+
+
+public class MenuActivity extends BankingAppBaseActivity {
```
Comment (helios66) [inline app/src/main/java/com/am/fdamilola/bankingapp/activities/MenuActivity.java]: Rename `BankingAppBaseActivity` to `TinyGameAppBaseActivity`

File: app/src/main/java/com/am/fdamilola/bankingapp/presenter/menuactivity/MenuActivityPresenter.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.am.fdamilola.bankingapp.presenter.menuactivity;
+
+/**
+ * Created by Princess on 02/05/2018.
+ */
```
Comment (helios66) [inline app/src/main/java/com/am/fdamilola/bankingapp/presenter/menuactivity/MenuActivityPresenter.java]: Always remove this header comment.

File: app/src/main/res/drawable/single_player_button.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                    <solid android:color="@color/SPButtonShadow" />
+                </shape>
+            </item>
+            <item android:bottom="4dp" android:left="0dp">
+                <shape
+                    xmlns:android="http://schemas.android.com/apk/res/android"
+                    android:shape="rectangle">
+
+                    <gradient android:startColor="@color/SPButtonGreenLight"
+                        android:endColor="@color/SPButtonShadow"
```
Comment (helios66) [inline app/src/main/res/drawable/single_player_button.xml]: `sp_button_shadow` for variables inside xml.
`aButton` camel case for variables inside Java.

File: app/src/main/res/layout-land/activity_menu.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/menu_bg"
+    tools:context="com.am.fdamilola.bankingapp.activities.MenuActivity"
+    tools:ignore="MissingPrefix">
+
+    <Button
+        android:id="@+id/single_player_button"
+        android:layout_width="354dp"
```
Comment (helios66) [inline app/src/main/res/layout-land/activity_menu.xml]: `@dimen/default_button_width` make this a dimension.

File: app/src/main/res/layout-land/activity_menu.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        android:textSize="40sp"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="50dp"/>
+
+    <Button
+        android:id="@+id/multi_player_button"
+        android:layout_width="354dp"
+        android:layout_height="70dp"
+        android:layout_centerHorizontal="true"
+        android:background="@drawable/multi_player_button"
```
Comment (helios66) [inline app/src/main/res/layout-land/activity_menu.xml]: Consider making the button a layout and using include layout.

File: app/src/main/res/values/colors.xml [hunk 6]
```diff
@@ -39,17 +39,19 @@
 
     <!--Main menu color palette -->
     <color name="SPButtonGreenLight">#53CB51</color>
-    <color name="SPButtonGreenDark">#174C16</color>
+    <color name="SPButtonGreenDark">#33174C16</color>
```
Comment (helios66) [inline app/src/main/res/values/colors.xml]: The naming convention for variables in xml.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1,4 +1,11 @@
 <resources>
     <string name="app_name">Color Roulette</string>
     <string name="play">Play</string>
+
+    <string name="title_activity_main_menu">MainMenuActivity</string>
```
Comment (helios66) [inline app/src/main/res/values/strings.xml]: Make variables alphabetical.

**Review by helios66 — CHANGES_REQUESTED** [review]
Fix the issues and I'll merge into the dev branch.

**Comment by princessdharmy** [comment]
@helios66 all changes have been made except the `include` button that could not be manipulated in the XML code

**Review by helios66 — APPROVED** [review]
This is dope!

**Pull request merged**